src/public/video.ts

```diff
@@ -50,6 +54,14 @@ export namespace video {
      * video format
      */
     format: VideoFrameFormat;
+    /**
+     * Synthesizing video frame based on raw frame or not
+     */
+    requireCameraStream?: boolean;
+    /**
+     * onnx model for audio inference
```

Comment:
Please be consistent with capitalization and also use language that is easier to understand. What is the onnx model? Is this a publicly defined concept?